server/src/sim.rs

```diff
@@ -51,13 +53,14 @@ pub struct Sim {
 impl Sim {
     pub fn new(cfg: Arc<SimConfig>, save: &save::Save) -> Self {
         let mut result = Self {
-            rng: SmallRng::from_entropy(),
             step: 0,
+            rng: SmallRng::from_entropy(),
             entity_ids: FxHashMap::default(),
             world: hecs::World::new(),
             graph: Graph::new(cfg.chunk_size),
             preloaded_voxel_data: FxHashMap::default(),
             spawns: Vec::new(),
+            pending_ticker_spawns: Vec::new(),
```

Comment:
As `pending_ticker_spawns` is only used in the `step` function, let's make it a local variable.

Reply:
Right, I don't know why I made that a field.